app/views/branches/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= render partial: 'validations' %>
```

Comment:
can we rename the partial to error_summary, please? I think describe better what is since the validations are done in the Branch model.